lib/prometheus_exporter/server/runner.rb

```diff
@@ -1,7 +1,6 @@
 # frozen_string_literal: true
 
 require_relative '../client'
-require_relative '../instrumentation/unicorn'
```

Comment:
Just to make sure.. no one else in this file needs anything from `'../instrumentation/unicorn'`, right?